mdpow/tests/test_equilibration_script.py

```diff
@@ -13,7 +13,7 @@ def setup(self):
         self.tmpdir = td.TempDir()
         self.old_path = os.getcwd()
         self.resources = os.path.join(
-            self.old_path, 'mdpow', 'tests', 'testing_resources')
+            self.old_path, 'testing_resources')
```

Comment:
I am not sure if this should be changed... unless I changed it in the working tests.

Reply:
It wasn't actually necessary, my testing configuration used the tests folder as the base dir giving ~/mdpow/tests/testing_recources/mdpow/tests/testing_recources so I changed, but later realized I could just use the project directory in my run configuration.

Reply:
Removed in 6679c73